src/network/networkd-conf.c

```diff
@@ -152,4 +161,11 @@ int config_parse_duid_rawdata(
         *dhcp_duid_len = duid_index;
 
         return 0;
+
+fail:
+        /* Ignore invalid DUID assignment */
+        *duid_type = _DUID_TYPE_INVALID;
+        *dhcp_duid_len = 0;
+        *dhcp_duid_type = 0;
+        return 0;
```

Comment:
I don't like this chunk. If we say "ignoring", we shouldn't set a value. In particular, if the config file has:
```
RawData=ab:bc:de
RawData=something-invalid
```
the value would be reset to `_DUID_TYPE_INVALID` unexpectedly.